### PR TITLE
Brief report

### PR DESCRIPTION
**Pull request by BrijeshKrishnan**

### CODE REVIEW

**Comment by BrijeshKrishnan** [comment]
Added html reporting for a brief reporting. This reporting can be used for presenting to any team for the quick wins

File: test/verify_path.py [hunk 1]
```diff
@@ -35,8 +35,12 @@ def verify_functional_test(self, new_text=False):
                 __data_recomend = pd.read_csv(TestResource.golden_recommendation_file_path)
 
             act_df_recomend = pd.read_csv(TestResource.recommendation_file_path)
+            act_html_report = pd.read_html(TestResource.brief_report_path)
+            ref_html_report = pd.read_html(TestResource.golden_brief_report_path)
             act_df_merged = pd.read_excel(TestResource.merged_file_path)
             act_df_duplicated = pd.read_excel(TestResource.duplicate_id_file_path)
+            self.assertEqual(act_html_report[0].replace(r'\\r', '', regex=True).values.tolist(),
```
Comment (dsvellal) [inline test/verify_path.py:42]: Thank you for adding the functional test case for this change!

File: test/test_cmd_log.py [hunk 2]
```diff
@@ -43,15 +43,31 @@ def test_colint(self):
...
 
+    def test_numrowcount(self):
+        """ Function to test the number of row count in the command line
+                correct and incorrect """
+        with self.assertRaises(SystemExit):
+            check_create_parser("-n", '1')
+        parsed = check_create_parser("--n", '1')
+        self.assertEqual(parsed.numrowcount, 1)
+
+    def test_numrowcount_default(self):
```
Comment (dsvellal) [inline test/test_cmd_log.py:54]: Great to see values being defaulted if none specified, and being tested via unit-tests!

File: build_scripts/dependencies_static_analysis_test_cov.py [hunk 3]
```diff
@@ -51,9 +51,8 @@ def check_md_linting():
     """
     function check the repo for any yml linting errors
     """
-    print("----TODO----- Markdown linting")
-    # call_subprocess("markdownlint *.md ")
-    # print("Stage linting md files -- COMPLETED & PASSED  --")
+    call_subprocess("markdownlint *.md ")
```
Comment (dsvellal) [inline build_scripts/dependencies_static_analysis_test_cov.py:54]: Nice touch - good to do linting for markdown. 

Can you also consider doing spell-checks for markdowns?

Comment (BrijeshKrishnan) [reply build_scripts/dependencies_static_analysis_test_cov.py:63]: Spell check for markdown is done through the current Aspell. in place.

**Pull request merged**